home/navigation/Navigation.tsx

```diff
@@ -211,6 +212,18 @@ function TabNavigator(props: { theme: string }) {
           tabBarLabel: 'Profile',
         }}
       />
+      {Boolean(__DEV__) && (
```

Comment:
`__DEV__` should already be a boolean. If not, `&&` will treat it like one anyway

```suggestion
      {__DEV__ && (
```

Reply:
there are cases where this could cause a crash for rendering a number / text outside a text component, im not sure what __DEV__ comes through as so its just a safety measure

Reply:
ah you're right! i usually do the ternary pattern in RN to avoid that